### PR TITLE
IAI-5449: update run container volumes section

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
https://deploy-preview-2383--axway-open-docs.netlify.app/docs/apim_installation/apiportal_docker/docker_portal_run_image/#create-data-volumes-to-persist-data

### CODE REVIEW

**Review by dginevaAxway — COMMENTED** [review]
I know that restart will be done after upgrade also but @kvedenskii-axway @andreamussap do you think that maybe we need to make it explicit and say "restart or upgrade", instead only "restart" ? Or I am mistaken for upgrade?

**Comment by andreamussap** [comment]
> restart or upgrade"

I'm not sure I get your question. Are you asking whether we should change "Do not modify the content of the following folders, as they will be overwritten on container restart" to

"...as they will be overwritten after an upgrade **or on** container restart" (meaning, every time that a container is restarted)

or if it is implicit that the container is automatically restarted after an upgrade, you could say

"...as they will be overwritten after an upgrade"

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md [hunk 1]
```diff
@@ -382,12 +382,12 @@ The following list describes which API Portal assets you should store in a Dock
 * `/opt/axway/apiportal/htdoc/administrator/language` - Joomla! admin panel translations.
 * `/opt/axway/apiportal/htdoc/administrator/components/com_apiportal/assets/cert` - Certificates for API Manager.
 
-Do not modify the content of the following folders, as they will be overwritten during upgrade:
+Do not modify the content of the following folders, as they will be overwritten on container restart:
```
Comment (dginevaAxway) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md:385]: Basically I suggest to keep both **upgrade** and **restart** , in practice restart will happens after upgrade and some customers maigh search in the doc for upgrade section and miss this restrictions. Suggest to have this:
"Do not modify the content of the following folders, as they will be overwritten on container restart/upgrade:"

Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md:385]: (they will be overwritten on container restart/upgrade) --> ok, if this is how it works, I can make the change myself

Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md:385]: yes

**Pull request merged**